website_v2/gcloud-powershell-website-v2/www/static/app/referenceController.js

```diff
@@ -0,0 +1,7 @@
+﻿var app = angular.module('powershellSite');
+
+/** The controller that controls url parameters. **/
```

Comment:
Can you elaborate this?

Is this just used to expose the route params to the templates?